addons/mail/static/src/core/common/thread.xml

```diff
@@ -3,7 +3,7 @@
 
 <t t-name="mail.Thread">
     <t t-if="env.inChatter" t-call="mail.Thread.jumpPresent"/> <!-- chatter has its own scrollable, this ensures proper sticky showing -->
-    <div class="o-mail-Thread position-relative flex-grow-1 d-flex flex-column overflow-auto" t-att-class="{ 'pb-4': !state.showJumpPresent }" t-ref="messages" tabindex="-1">
+    <div class="o-mail-Thread position-relative flex-grow-1 d-flex flex-column overflow-auto" t-att-class="{ 'pb-4': !state.showJumpPresent and !props.showJumpPresent }" t-ref="messages" tabindex="-1">
```

Comment:
`and props.showJumpPresent` (not `!`).

Nitpicking but it feels easier to read with `props.showJumpPresent` before `state.showJumpPresent`. The former is a configuration, the other is dynamic state, it feels more natural config before dynamic state in the conditions